packages/core/src/integration.ts

```diff
@@ -5,9 +5,9 @@ import { logger } from '@sentry/utils';
 export const installedIntegrations: string[] = [];
 
 /** Map of integrations assigned to a client */
-export interface IntegrationIndex {
+export type IntegrationIndex = {
   [key: string]: Integration;
-}
+} & { initialized?: boolean };
```

Comment:
Why an intersection type?

```ts
export type IntegrationIndex = {
   [key: string]: Integration;
   initialized?: boolean;
}
```

Reply:
Because it makes my linter all kinds of mad:

![image](https://user-images.githubusercontent.com/14812505/128038224-a8bd394b-cac0-4d85-ba25-a9c4f7e42530.png)

Same if it's an interface:

![image](https://user-images.githubusercontent.com/14812505/128038364-36477462-24a2-4ba2-9463-61dab3ad83a3.png)

🎺

Reply:
Note that this now applies to https://github.com/getsentry/sentry-javascript/pull/3862 rather than this PR.